packages/fxa-auth-server/README.md

```diff
@@ -147,6 +147,64 @@ This saves the HTML template into `/templates`. Then make changes to the `.txt`
 After updating a string in one of the templates in `./mailer/templates` you'll need to extract the strings.
 Follow the instructions at [mozilla/fxa-content-server-l10n](https://github.com/mozilla/fxa-content-server-l10n#string-extraction).
 
+### 2021 Template Updates
+
+In 2021, FxA began converting the email templating system from Mustache, inline CSS, and GetText to a modernized stack using [MJML](https://mjml.io/), [EJS](https://ejs.co/), [Fluent](https://github.com/projectfluent/fluent.js/tree/master/fluent-dom), and [Storybook](https://github.com/storybookjs/storybook/tree/main/app/html) ([see the ADR](https://github.com/mozilla/fxa/blob/main/docs/adr/0024-upgrade-templating-toolset-of-auth-server-emails.md)). Until each template has been converted and verified by QA, the old system described above will be used for emails in production.
+
+#### MJML Feature Flag for Testing
+
+The auth-server has an `mjml` configuration value that designates which email templates have been converted to the new stack ( `mjml.templates` array) and controls which user emails will receive those templates ( `mjml.enabledEmailAddress` regex). The auth-server exposes a feature flag method to check these values and send the correct rendered template accordingly (old or new) which allows for testing across environments.
+
+This flag and logic around it can be removed, as well as old templates and documentation, once all templates have been converted, verified by QA, and tested in production.
+
+#### MJML and EJS
+
+- HTML email has a lot of quirks - MJML shifts the burden of maintaining solutions for these off of FxA engineers
+- Using EJS with MJML helps us include more logic in templates thereby making conditional rendering possible without creating any additional helper methods
+- A small example of how template variables are being passed down from the mailer object and consumed in the templates is shown below:
+
+```js
+const templateValues = {
+  buttonText: 'Sync another device',
+};
+const localized = this.templates.render(
+  message.template, // template name
+  message.layout || 'fxa', // template layout
+  templateValues
+);
+const button = `
+  <mj-include path="./lib/senders/emails/css/button/index.css" type="css" css-inline="inline" />
+  <mj-section>
+    <mj-column>
+      <mj-button css-class="primary-button"><%= buttonText %></mj-button>
+    </mj-column>
+  </mj-section>
+`;
+```
+
+- The emails for which the mjml feature flag is enabled can be rendered to disk using the `yarn write-emails` command
```

Comment:
Is this supposed to be `yarn write-templates`? I'm sort of confused by the similar naming between old and new templates. Maybe we can have another section to point out which commands are for the old/current templates along with where the old/current and new templates are located since if someone else goes to pick up any of this work it might be confusing for them too to get started.